tf2onnx/onnx_opset/tensor.py

```diff
@@ -2453,14 +2661,28 @@ def compute_out_shape(k0_k1_same=False):
             if consumer.type == 'Identity':
                 ctx.set_shape(consumer.output[0], shapes)
 
+    @classmethod
+    def version_12(cls, ctx, node, **kwargs):
+        cls.any_version_after12(12, ctx, node, **kwargs)
+
+    @classmethod
+    def version_13(cls, ctx, node, **kwargs):
+        # Parameters moved to inputs for operator Squeeze, Unsqueeze.
+        cls.any_version_after12(13, ctx, node, **kwargs)
+
 
 @tf_op(["MatrixDiag", "MatrixDiagV2", "MatrixDiagV3"])
 class MatrixDiag:
     @classmethod
-    def version_12(cls, ctx, node, **kwargs):
+    def any_version(cls, opset, ctx, node, **kwargs):
         # Assemble MatrixDiagV3 by ReverseSequence
         argc = len(node.input)
 
+        if opset >= 13:
+            squeeze_axes0 = GraphBuilder(ctx).convert_to_input([0], "const_axes", is_optional=True, dtype=np.int64)
```

Comment:
Can you use make_const here?  I'd like to keep convert_to_input internal to GraphBuilder